otx/algorithms/action/adapters/mmaction/task.py

```diff
@@ -295,9 +295,18 @@ def _train_model(
 
         validate = bool(cfg.data.get("val", None))
 
-        if self._hyperparams.learning_parameters.auto_decrease_batch_size:
+        if (
+            self._hyperparams.learning_parameters.auto_decrease_batch_size
+            or self._hyperparams.learning_parameters.auto_adapt_batch_size
```

Comment:
I agree that these two parameter from cli is better to be combined. I suggest to make a string argument as ('maximum', 'auto') rather than ('decrease', 'auto'). But I suddenly got curious, if merge these config, what happens if user put the runnable batch size and enabled this with 'decrease'? Is batch size kept or automatically returned with the large batch or maximum batch size?

Reply:
If user set it as `decrease` then just checking defulat batch size is runnable with current GPU memory. it try to find a runnable batch size only when default batch size isn't runnable.